specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json

```diff
@@ -88,6 +88,60 @@
         }
       }
     },
+    "HealthcareDicomImageCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.HealthcareApis.DicomImageCreated event.",
+      "type": "object",
+      "properties": {
+        "dicomImageStudyInstanceUid": {
+          "description": "Unique identifier for the Study",
+          "type": "string"
+        },
+        "dicomImageSeriesInstanceUid": {
+          "description": "Unique identifier for the Series",
+          "type": "string"
+        },
+        "dicomImageSopInstanceUid": {
+          "description": "Unique identifier for the DICOM Image",
+          "type": "string"
+        },
+        "dicomServiceHostName": {
+          "description": "Domain name of the DICOM account for this image.",
+          "type": "string"
+        },
+        "dicomServiceSequence": {
+          "description": "Sequence number of the DICOM Service within Azure Health Data Services. It is unique for every image creation and deletion within the service.",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "HealthcareDicomImageDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.HealthcareApis.DicomImageDeleted event.",
+      "type": "object",
+      "properties": {
+        "dicomImageStudyInstanceUid": {
```

Comment:
I think for the FHIR events, we included Fhir in all of the names to disambiguate the resourceId property in particular from the Azure Resource ID concept. I don't think we have the same issue here, so I would suggest dropping "dicom" from each property name.

Reply:
Makes sense. Removed the dicom prefix